### PR TITLE
feat: storage service frontend

### PR DESCRIPTION
**Pull request by Zeng1998**
https://user-images.githubusercontent.com/31177490/218513874-a278f902-1f09-40f7-a307-7b1f62a6007e.mp4

### CODE REVIEW

File: api/system_setting.go [hunk 1]
```diff
@@ -25,8 +25,8 @@ const (
 	SystemSettingAdditionalScriptName SystemSettingName = "additionalScript"
 	// SystemSettingCustomizedProfileName is the key type of customized server profile.
 	SystemSettingCustomizedProfileName SystemSettingName = "customizedProfile"
-	// SystemSettingStorageServiceName is the key type of sotrage service name.
-	SystemSettingStorageServiceName SystemSettingName = "storageServiceName"
+	// SystemSettingStorageServiceID is the key type of sotrage service name.
```
Comment (boojack) [inline api/system_setting.go]: name -> id

File: server/resource.go [hunk 2]
```diff
@@ -86,12 +86,25 @@ func (s *Server) registerResourceRoutes(g *echo.Group) {
...
 		systemSetting, err := s.Store.FindSystemSetting(ctx, &api.SystemSettingFind{Name: &systemSettingStorageServiceName})
 		if err != nil && common.ErrorCode(err) != common.NotFound {
 			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to find storage").SetInternal(err)
 		}
-		if common.ErrorCode(err) == common.NotFound || systemSetting.Value == "" {
+		storeLocal := false
+		if common.ErrorCode(err) == common.NotFound {
+			storeLocal = true
+		} else {
+			var value interface{}
```
Comment (boojack) [inline server/resource.go:98]: interface{} -> int

File: web/src/components/SettingDialog.tsx [hunk 3]
```diff
@@ -52,6 +53,12 @@ const SettingDialog: React.FC<Props> = (props: Props) => {
           >
             <span className="icon-text">🏟</span> {t("setting.preference")}
           </span>
+          <span
```
Comment (boojack) [inline web/src/components/SettingDialog.tsx:56]: Only host users have the accessibility.

**Review by boojack — APPROVED** [review]
LGTM overall

**Comment by Zeng1998** [comment]
Please check the latest commit.

**Pull request merged**